Yafc.Model/Serialization/UndoSystem.cs

```diff
@@ -91,13 +96,9 @@ public void PerformRedo() {
         undo.Push(redo.Pop().Restore(++version));
     }
 
-    public void RecordChange() {
-        ++version;
-    }
+    public void RecordChange() => ++version;
```

Comment:
Is there any reason this is not `version++`? I never use the `++` before the variable unless i need it to specifically return the increased value. This function is `void` however.

Reply:
I just left it as it was before the wrap. Or, to be more precise, I didn't pay attention to that. 
I think your reasoning is solid. If you want to address this part, feel free to add a fixup with the change.

The only thing that I'd suggest is to check the usage of this function, just in case.

Reply:
I checked, it's safe to change here. There are a few other instances of `++version` in the code that we cannot safely change. Added a fixup commit.